src/modules/get_selfies_to_review/app/get_selfies_to_review_controller.py

```diff
@@ -0,0 +1,43 @@
+from src.shared.domain.entities.student import Student
+from src.shared.domain.entities.selfie import Selfie
+from src.shared.helpers.errors.domain_errors import EntityError
+from src.shared.helpers.errors.usecase_errors import NoItemsFound
+from src.shared.helpers.errors.controller_errors import MissingParameters
+from src.shared.helpers.http.http_models import OK, BadRequest, HttpRequest, HttpResponse, InternalServerError, NotFound
+from .get_selfies_to_review_usecase import GetSelfiesToReviewUsecase
+from .get_selfies_to_review_viewmodel import GetSelfiesToReviewViewmodel
+
+class GetSelfiesToReviewController:
+    def __init__(self, usecase: GetSelfiesToReviewUsecase):
+        self.getSelfiesToReviewUsecase = usecase
+
+    def __call__(self, request: HttpRequest) -> HttpResponse:
+        try:
+            if request.query_params.get('reviewerRa') is None:
+                raise MissingParameters('reviewerRa')
+
+            if request.query_params.get('nSelfies') != None:
+                if type(request.query_params.get('nSelfies')) != str:
+                    raise EntityError('nSelfies')
+                elif not request.query_params.get('nSelfies').isdecimal():
+                    raise EntityError('nSelfies')
+
+            reviews, reviewer = self.getSelfiesToReviewUsecase(
+                nSelfies=int(request.query_params.get('nSelfies')) if request.query_params.get('nSelfies') != None else 10,
```

Comment:
eu acho que essa validação se for None deveria existir no usecase, pois é uma regra de negócio a base ser 10 selfies, não lógica do controller, senão não serviu para nada o = 10 na chamada da função no usecase porque o controller tá forçando o 10. Basicamente se você quiser mudar lá pra 11, aq tbm vai ter que mudar...